pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -222,7 +222,7 @@ func nodesWherePreemptionMightHelp(nodes []*framework.NodeInfo, m framework.Node
 		name := node.Node().Name
 		// We reply on the status by each plugin - 'Unschedulable' or 'UnschedulableAndUnresolvable'
 		// to determine whether preemption may help or not on the node.
-		if m[name].Code() == framework.UnschedulableAndUnresolvable {
+		if s, ok := m[name]; !ok || s.Code() == framework.UnschedulableAndUnresolvable {
```

Comment:
`m[code].Code()` will return `Success` when `m[code]` is nil, so this is functionally the same as your change.